roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -9,14 +9,35 @@
 # - openshift_release
 
 # tasks file for openshift_node_upgrade
+
+- name: Stop node and openvswitch services
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+  - "{{ openshift.common.service_type }}-node"
+  - openvswitch
+  failed_when: false
+
+- name: Stop additional containerized services
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+    - "{{ openshift.common.service_type }}-master"
+    - "{{ openshift.common.service_type }}-master-api"
+    - "{{ openshift.common.service_type }}-master-controllers"
+    - etcd_container
+  failed_when: false
+  when: openshift.common.is_containerized | bool
+
 - include: docker/upgrade.yml
   vars:
     # We will restart Docker ourselves after everything is ready:
     skip_docker_restart: True
```

Comment:
`skip_docker_restart` is not used anywhere in the ``docker/upgrade.yml`` so it can be removed